maintenance_sign_oca/models/maintenance_equipment.py

```diff
@@ -70,11 +70,18 @@ def create(self, vals_list):
             res._process_generate_sign_oca_request({})
         return res
 
+    def _sign_oca_request_get_user_fields(self):
+        """Fields to be taken into account when trying to create a sign request.
+        We don't need to take into account only the owner_user_id field because
+        if you have installed hr_maintenance module is a compute field and the
+        employee_id field will be taken into account."""
+        return [fname for fname in ("owner_user_id", "employee_id") if fname in self]
```

Comment:
Why the if? Can it be directly?

```suggestion
        return ["owner_user_id", "employee_id"]
```